### DISCUSSION TITLE
A better development pipeline?

### DISCUSSION THREAD

**Discussion by kestrel-x86** [micropython/micropython#13381] [ESP8266]
I know my way around python but I'm far from an expert, and the project I'm working on now has a web ui so I'm doing frontend stuff that I'm only kind of familiar with.

This means I'm making a lot of smallish adjustments, uploading to the board, testing, and repeating this many times.

A lot of time is consumed by just moving files from my computer to my board, and I have concerns (and maybe I'm a little paranoid) about unnecessary wear on the flash.

What's the most efficient way to work with micropython like this?

**Comment by andrewleech** [comment]
Take a look at [mpremote mount](https://docs.micropython.org/en/latest/reference/mpremote.html#mpremote-command-mount)

**Comment by bulletmark** [comment]
I suggest you read [this recent thread](https://github.com/orgs/micropython/discussions/13190) and continue with questions/discussion there.
